netci.groovy

```diff
@@ -6,6 +6,8 @@ def project = GithubProject
 
 // Globals
 
+// List of all CI OS
+def allOSs = ['Ubuntu', 'Debian8.2', 'OSX', 'FreeBSD', 'CentOS7.1', 'OpenSUSE13.2', 'Windows 10', 'Windows 7', 'Windows_NT']
```

Comment:
This is not used anywhere currently, but will be used when all OSes are supported in Outerloop. Will remove the def Outerloop then. Sorry your comment didnt update on my client.. Saw this only now.